src/agent.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { Finding, FindingSeverity, FindingType, HandleTransaction, createTransactionEvent } from "forta-agent";
+import agent, { AGENT_REGISTRY_ADDRESS, CREATE_BOT_FUNCTION, DEPLOYER_ADDRESS } from "./agent";
+
+describe("bot created by deployer bot", () => {
+  let handleTransaction: HandleTransaction;
+  const mockTxEventDeployer = createTransactionEvent({
```

Comment:
we use our own library for creating these test transaction events. it's here https://github.com/NethermindEth/general-agents-module
I suggest you to study it for a while and use it in this bot. it's a good practice to learn.